### PR TITLE
UI: Improve ui with boostrap-vue #116

### PR DESCRIPTION
**Pull request by Pols95**
- new sidebar
- move tables to bootstrap-vue ones (featuring pagination, columns sorting etc)
- other minor ui bootstrap-vue improves

### CODE REVIEW

**Comment by eolivelli** [comment]
@mino181295  please take a look

**Comment by eolivelli** [comment]
@nicoloboschi  PTAL

File: carapace-ui/src/main/webapp/package.json [hunk 1]
```diff
@@ -9,11 +9,15 @@
...
   "dependencies": {
     "bootstrap": "4.3.1",
-    "bootstrap-vue": "2.0.0-rc.11",
+    "bootstrap-vue": "2.0.4",
     "moment": "2.24.0",
     "vue": "2.6.10",
     "vue-cli": "2.9.6",
-    "vue-router": "3.0.6"
+    "vue-router": "3.0.6",
+    "@fortawesome/fontawesome-svg-core": "^1.2.19",
```
Comment (nicoloboschi) [inline carapace-ui/src/main/webapp/package.json]: It is better to use fixed versions of dependencies

File: carapace-ui/src/main/webapp/src/App.vue [hunk 2]
```diff
@@ -1,113 +1,116 @@
...
+    },
+    data() {
+        return {
+            peerId: ""
+        };
+    },
+    created() {
+        var url = "/api/cluster/localpeer";
+        var self = this;
+        doGet(url, data => {
```
Comment (nicoloboschi) [inline carapace-ui/src/main/webapp/src/App.vue]: Using lamba functions "this" keyword has the same value as outside of the function, so in this case self === this

File: carapace-ui/src/main/webapp/src/components/CacheStatus.vue [hunk 3]
```diff
@@ -1,101 +1,117 @@
...
-            })
+            var self = this;
 
+            doGet(url, data => {
+                self.status = data;
+            });
+
+            doGet("/api/cache/inspect", data => {
+                data.forEach(it => {
+                    it.key = it.host + it.uri;
```
Comment (nicoloboschi) [inline carapace-ui/src/main/webapp/src/components/CacheStatus.vue]: this.getKey(it) ?

**Pull request merged**